api/delays.go

```diff
@@ -49,3 +50,75 @@ func (e *Env) GetDelays(w http.ResponseWriter, r *http.Request) {
 
 	w.Write(rb)
 }
+
+func (e *Env) GetSubedDelays(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	tk, ok := ctx.Value("Token").(data.Token)
+
+	if !ok {
+		log.Printf("Token from context not of type ctx: %v", tk)
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(output.JSON500Response))
+		return
+	}
+
+	del, err := e.ObjStore.Get("delays")
+
+	if err != nil {
+		log.Println(err)
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(output.JSON500Response))
+		return
+	}
+
+	var od delays.Out
+
+	if err := json.Unmarshal(del, &od); err != nil {
+		log.Println(fmt.Errorf("api - Delays: failed to parse json: %v", err))
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(output.JSON500Response))
+		return
+	}
+
+	subs, err := e.Subscriptions.GetAll(tk.UserID)
+
+	if err != nil {
+		log.Println(err)
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(output.JSON500Response))
+		return
+	}
+
+	var osubs []delays.OutTrip
+
```

Comment:
Maybe add a comment to explain this loop. It's purpose isn't immediately obvious